coordinator/pom.xml

```diff
@@ -93,18 +93,21 @@
       -->
     <netty.version>4.1.75.Final</netty.version>
     <netty-boringssl.version>2.0.51.Final</netty-boringssl.version>
-    <jacoco.version>0.8.8</jacoco.version>
+    <jacoco.version>0.8.10</jacoco.version>
```

Comment:
Was necessary to prevent lots of noise about failed instrumentation (although I don't think we actually use code coverage reports for anything).